pyintegration/tests/test_calendar.py

```diff
@@ -75,14 +93,8 @@ def test_business_schedule_business_day(self):
 
     def test_business_calendar(self):
         default_calendar = BusinessCalendar()
-        self.assertIsNotNone(default_calendar.name)
-        self.assertIsNotNone(default_calendar.current_day)
-        self.assertIsNotNone(default_calendar.previous_day)
-        self.assertIsNotNone(default_calendar.next_day)
-        self.assertIsNotNone(default_calendar.day_of_week.name)
-        self.assertEqual(default_calendar.time_zone, get_server_timezone())
-        self.assertIn(default_calendar.is_business_day, {True, False})
```

Comment:
I think some of these tests got lost.  Are we testing all of this still?

Reply:
yes, but most of them are now methods which take a single date str.